k8s/beacon-api/k8s-aks/airqo-beacon-api/templates/deployment.yaml

```
@@ -0,0 +1,48 @@
+                name: {{ .Values.app.k8sSecret }}   # ExternalSecret created per microservice
+          {{- with .Values.volumeMounts }}
+          volumeMounts:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+      {{- with .Values.volumes }}
+      volumes:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)